### DISCUSSION TITLE
Removing Employee Card Files Tab

### DISCUSSION THREAD

**Discussion by MikeH350** [pnp/sp-dev-fx-webparts#3641] [Q&A]
Hi, 
Is there a way to remove the files tab when a person's profile card is selected so their files can't be viewed? 

![employeeCardImg](https://user-images.githubusercontent.com/116731568/228369161-5ea638ec-678b-4252-b0a1-f36d85b77abb.png)

**Comment by bcameron1231** [comment]
These files are permission trimmed. It shows files that you have access to see (you shared with them), or files that the user has shared with you. 

It's the same as if you went to OneDrive to see the shared files. You aren't seeing anything you shouldn't see.

**Reply by MikeH350** [reply]
I'm aware you can only see files you have been given permission to see, I'm wondering if there's a tenant setting or a way to disable this tab for profile cards.

**Reply by bcameron1231** [reply]
No, I do not believe this is possible.
